src/YieldDisburser.sol

```diff
@@ -197,45 +164,70 @@ contract YieldDisburser is OwnableUpgradeable {
      * @param _start Start time of the period to return the voting power for
      * @param _end End time of the period to return the voting power for
      * @param _account Address of user to return the voting power for
-     * @return uint256 Voting power of the specified user at the specified period of time
+     * @return votingPowerTotal  Voting power of the specified user at the specified period of time
      */
-    function getVotingPowerForPeriod(uint256 _start, uint256 _end, address _account) external view returns (uint256) {
+    function getVotingPowerForPeriod(uint256 _start, uint256 _end, address _account)
+        external
+        view
+        returns (uint256 votingPowerTotal)
```

Comment:
return variable should be unnamed to be consistent with all the other return variables

Reply:
resolved with 16d9641

Reply:
addressed with a11a89d